src/Module.ts

```diff
@@ -264,8 +267,9 @@ export default class Module {
 			getReexports: this.getReexports.bind(this),
 			getModuleExecIndex: () => this.execIndex,
 			getModuleName: this.basename.bind(this),
-			includeNamespace: this.includeNamespace.bind(this),
 			imports: this.imports,
+			includeNamespace: this.includeNamespace.bind(this),
+			inFunction: false,
```

Comment:
While the context is a good place for the `usesTopLevelAwait` flag, which describes the whole module and will only every change once from `false` to `true`, it is a very bad place for the `inFunction` flag. Cf. my other comment at FunctionNode for some reasoning and ideas for a better approach.